### PR TITLE
Update Applications.md

### PR DESCRIPTION
**Pull request by arrmo**
Add details of how to get Wireguard working on Ubuntu (as the user, Debian-snmp)

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Extensions/Applications.md [hunk 1]
```diff
@@ -2759,6 +2759,11 @@ b.) "public_key_to_arbitrary_name" - A dictionary to convert between the publick
 
 5. Restart snmpd.
 
+On Ubuntu, snmpd runs as Debian-snmp, and as a result cannot "directly" access or run wg. BUT - if you add this user (Debian-snmp) to visudo, and allow `bash` to be run, then you can update the entry in snmpd.conf to be,
```
Comment (Jellyfrog) [inline doc/Extensions/Applications.md:2762]: That is a bigger security risk than the one already documented in the top of the file.

Comment (Jellyfrog) [inline doc/Extensions/Applications.md:2762]: That is a bigger security risk than the one already documented in the top of the file.

**Review by Jellyfrog — CHANGES_REQUESTED** [review]

File: doc/Extensions/Applications.md (hunk 1 above)
Comment (arrmo) [reply doc/Extensions/Applications.md:2762]: Understand, but without this - it won't run / work on Ubuntu 😉 

Thanks!

Comment (murrant) [reply doc/Extensions/Applications.md:2762]: Would be much better if it could be restricted to "wg show all dump"

**Pull request closed without merging**